injectproxy/alertgroups.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package injectproxy
+
+import (
+	"net/http"
+)
+
+func (r *routes) alertGroups(w http.ResponseWriter, req *http.Request) {
```

Comment:
since `alertGroups` and `listSilences` we can have a single method (maybe `enforceFilterParameter()`?).

Reply:
Does it not make sense for it to be different methods and have them call into a common method?  This common method can have all the common code.  It makes the separation  of responsibilities clear

Reply:
Done